deployment/templates/fluentd.yaml

```diff
@@ -58,20 +44,15 @@ objects:
             component: "fluentd"
         spec:
           serviceAccountName: aggregated-logging-fluentd
+          nodeSelector:
+            logging-infra: fluentd
           containers:
           - name: fluentd-elasticsearch
-            image: logging-fluentd
+            image: ${IMAGE_PREFIX}logging-fluentd
```

Comment:
I need to verify further, but I did notice that I had issues with the daemonset finding the image when I was using a dev-build unless I specified an Image Prefix.